.gitignore

```diff
@@ -1,20 +1,281 @@
-.DS_Store
```

Comment:
let's do one change per commit so as to have separate conversations about separate changes.